resolvers/node/resolution.rs

```diff
@@ -63,6 +64,35 @@ fn conditions_from_resolution_mode(
   }
 }
 
+pub struct ConditionsFromResolutionMode {
+  pub func: Box<
+    dyn Fn(ResolutionMode) -> &'static [&'static str] + Send + Sync + 'static,
+  >,
+}
+
+impl Debug for ConditionsFromResolutionMode {
+  fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+    f.debug_struct("ConditionsFromResolutionMode").finish()
+  }
+}
+
+impl ConditionsFromResolutionMode {
+  pub fn new(
+    func: Option<
```

Comment:
Maybe this should just accept the Box and then have a default impl of `ConditionsFromResolutionMode` that sets it to `None`?

Reply:
Done